src/libos/crates/host-socket/src/stream/states/connected/mod.rs

```diff
@@ -65,6 +65,14 @@ impl<A: Addr + 'static, R: Runtime> ConnectedStream<A, R> {
         Ok(())
     }
 
+    pub fn set_closed(&self) {
```

Comment:
When closing the socket, it is need to prevent submitting send/recv requests and discard kernel buffer.